dvf.libraries.yml

```diff
@@ -19,7 +19,7 @@ d3:
     url: https://github.com/d3/d3/blob/master/LICENSE
     gpl-compatible: true
```

Comment:
Now's probably a good time to address

Reply:
@tobybellwood good point! @rob-duplock could you please update this to `name: BSD-3`? Thanks